packages/google_sign_in/google_sign_in/example/lib/main.dart

```diff
@@ -33,31 +33,31 @@ class SignInDemo extends StatefulWidget {
 }
 
 class SignInDemoState extends State<SignInDemo> {
-  GoogleSignInAccount _currentUser;
-  String _contactText;
+  GoogleSignInAccount? _currentUser;
+  String? _contactText;
```

Comment:
super optional nits: default to empty String. 😝

Reply:
Done